lib/plugin.c

```diff
@@ -481,6 +481,12 @@ plugin_free_candidate_modules(GlobalConfig *cfg)
 }
 
 void
+plugin_get_module_path(FILE *out)
+{
+    fprintf(out, "Module path: %s\n", module_path);
+}
+
```

Comment:
also, the module_path global is pretty sure to be at the wrong location in mainloop.c and should probably be pushed down to plugin.c, but that's a separate issue.